README.md

```diff
@@ -14,6 +14,14 @@ You can find us on Twitter [@MozillaReality](https://twitter.com/mozillareality)
 git clone git@github.com:MozillaReality/FirefoxReality.git
 ```
 
+*Clone the third_party repo*
+
+If you're developing for the Oculus, Snapdragon VR, or the VIVE, you need to clone the repo with third party SDK files.
+
+```bash
+git clone git@github.com:MozillaReality/FirefoxReality-android-third-party.git third_party
```

Comment:
This is only available to Mozilla employees. That was why it wasn't included in the README. What needs to happen is this repository needs to be restructured to match the SDKs and then build variables added so that users can down load the SDKs from the official developer sites and then set the variables to point to the installed location. We just haven't had the cycles to get it done yet.

Reply:
Not having it in the docs at all makes it harder for Mozilla employees who are not involved on a day-to-day basis.

Reply:
Done, I also provided instructions for those without access to the repo (but with access to the SDKs)